app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable.phtml

```diff
@@ -10,6 +10,7 @@
 <?php
```

Comment:
Please remove not needed closing and opening PHP tags

Reply:
fixed